tests/sqllogictests/suites/tpcds/spill.test

```diff
@@ -29,6 +29,23 @@ set window_partition_spilling_bytes_threshold_per_proc = 1;
 # TPC-DS TEST
 include ./queries.test
 
+statement ok
+set max_block_size = 2;
```

Comment:
In test better set 1.

Reply:
If it is 1, the bug cannot be reproduced.